templates/repo.local.erb

```diff
@@ -29,6 +29,12 @@ CVMFS_CACHE_BASE='<%= @cvmfs_cache_base %>'
 <% if @cvmfs_claim_ownership and  @cvmfs_claim_ownership != '' -%>
 CVMFS_CLAIM_OWNERSHIP='<%= @cvmfs_claim_ownership %>'
 <% end -%>
+<% if @cvmfs_uid_map and !@cvmfs_uid_map.empty? -%>
```

Comment:
This can be simplified if default is undef.

Reply:
Indeed, done :-). 
Note that in general I tend not to use `undef` defaults since Foreman can not specify `undef` to any class, but since usually profile classes are used with Foreman, that is not an actual issue most of the time.